eth/src/main/java/contract/RelayRegistry.java

```diff
@@ -86,7 +86,7 @@ protected RelayRegistry(String contractAddress, Web3j web3j, TransactionManager
 
```

Comment:
Can we modify this file? I know that it was generated.

Reply:
Could you be more specific on that? What exact changes would you like to apply?

Reply:
I think you should ask @Alexey-N-Chernyshov .

Reply:
Anton is correct. All java/contract/*.java files are auto-generated code that shouldn't be modified.

Reply:
Done